protocol/src/protocol.ts

```diff
@@ -665,6 +679,11 @@ export interface _ServerCapabilities<T = any> {
 	 */
 	executeCommandProvider?: ExecuteCommandOptions;
 
+	/**
+	 * The server provides Call Hierarchy support.
+	 */
```

Comment:
Can we add an `@since 3.16.0` tag here too?

Reply:
None of the other fields contain `@since` so I didn't add it here either.

Reply:
I was sloppy on this. We should add @since 3.16 here. I tried to do this for all new features as well.